WordPress/Classes/ViewRelated/Tools/SettingsTextViewController.m

```diff
@@ -228,13 +281,16 @@ - (void)updateModeSettings:(SettingsTextModes)newMode
 {
     BOOL requiresSecureTextEntry = NO;
     UIKeyboardType keyboardType = UIKeyboardTypeDefault;
+    UITextAutocapitalizationType autocapitalizationType = UITextAutocapitalizationTypeWords;
```

Comment:
You're right, the default is actually `UITextAutocapitalizationTypeSentences`, i'll switch to that.

This was added because... email addresses were being capitalized. Just handling the default behavior as well.